### ISSUE TITLE
Manual triggering of an import of an existing file via API in GCP - result is null

### ISSUE THREAD

**Issue by mbeilin** [datopian/aircan#46]
When a user manually triggering an import of an existing file via API in GCP the result is always `null`.
Sample API call:
```
curl -X POST \
-H 'Authorization: YOUR_CKAN_API_KEY' \
-d '{"resource_id": "RESOURCE_ID"}' \
http://CKAN_HOST/api/action/datapusher_submit
```
In case we are running in GCP env (`CKAN__AIRFLOW__CLOUD=GCP`), the DAG is triggered and
the response is:
```
{
    "help": "http://ckan:5000/api/3/action/help_show?name=datapusher_submit",
    "success": true,
    "result": null
}
```

On the other hand, running the same API call in local env (`CKAN__AIRFLOW__CLOUD=LOCAL`),
we are getting a proper response:
```
{
    "help": "http://ckan:5000/api/3/action/help_show?name=datapusher_submit",
    "success": true,
    "result": {
        "message": "Created <DagRun ckan_api_load_multiple_steps @ 2020-07-06 07:27:15+00:00: manual__2020-07-06T07:27:15+00:00, externally triggered: True>",
        "execution_date": "2020-07-06T07:27:15+00:00",
        "run_id": "manual__2020-07-06T07:27:15+00:00"
    }
}
```

## Acceptance

- [ ] Manually triggering an import of an existing file via API in GCP and in Local envs both bring a proper response.

## Tasks

- [ ]  Check the returned value from function  [`invoke_gcp`](https://github.com/datopian/ckanext-aircan/blob/master/ckanext/aircan_connector/action.py#L63)

**Comment by hannelita** [comment]
This will be addressed on #30; this is not done yet

**Comment by hannelita** [comment]
Fixed on #62
